static/js/controllers/configuration-upgrade.js

```diff
@@ -0,0 +1,110 @@
+var PouchDB = require('pouchdb-browser');
+
+// REVIEWER: are we OK hard-coding these two concepts? If not, where should
+// we put this? We don't have access to env vars in medic-webapp.
+var IS_PROD_URL = /^https:\/\/[^.]+.app.medicmobile.org\//,
```

Comment:
So my preference for this is at some point we move this code into horti completely, and like you say parametrise this. For now I'm going to leave it though.